### PR TITLE
clam*: add Thai translation

### PR DESCRIPTION
**Pull request by nontw**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.th/linux/clamav.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# clamav
```
Comment (navarroaxel) [inline pages.th/linux/clamav.md:1]: Oh, the English page is not valid. This should be an alias page, or similar to https://github.com/tldr-pages/tldr/blob/main/pages/common/transmission.md

Comment (nontw) [reply pages.th/linux/clamav.md:1]: Oh ok .. I just saw that too.  I just changed the original English content of clamav to be an alias of clamscan and adjusted the Thai page of clamav accordingly.

File: pages.th/common/clamdscan.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# clamdscan
+
+> โปรแกรมแสกนหาไวรัสบนคอมมานด์ไลน์ โดยใช้โปรแกรมพื้นหลัง ClamAV
+> ข้อมูลเพิ่มเติม: <https://www.clamav.net>.
+
+- แสกนข้อบกพร่องของไฟล์หรือไฟล์ในไดเรคทอรี
+
+`clamdscan {{path/to/file_or_directory}}`
+
+- แสกนข้อมูลจากสตรีมอินพุทมาตรฐาน
```
Comment (kbdharun) [inline pages.th/common/clamdscan.md:10]:
```suggestion
- แสกนข้อมูลจากสตรีมอินพุทมาตรฐาน:
```

File: pages.th/common/clamdscan.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`{{command}} | clamdscan -`
+
+- แสกนไดเรคทอรีปัจจุบันแล้วแสดงผลไฟล์ที่ตรวจพบความบกพร่อง
+
+`clamdscan --infected`
+
+- ส่งผลการแสกนไปยังไฟล์ที่ระบุ
```
Comment (kbdharun) [inline pages.th/common/clamdscan.md:18]:
```suggestion
- ส่งผลการแสกนไปยังไฟล์ที่ระบุ:
```

File: pages.th/common/clamdscan.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# clamdscan
+
+> โปรแกรมแสกนหาไวรัสบนคอมมานด์ไลน์ โดยใช้โปรแกรมพื้นหลัง ClamAV
+> ข้อมูลเพิ่มเติม: <https://www.clamav.net>.
+
+- แสกนข้อบกพร่องของไฟล์หรือไฟล์ในไดเรคทอรี
+
+`clamdscan {{path/to/file_or_directory}}`
```
Comment (kbdharun) [inline pages.th/common/clamdscan.md:8]: Translate this path, and other command arguments inside `{{}}` throughout the pages.

File: pages.th/common/clamdscan.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- ย้ายไฟล์ที่พบการติดไวรัสไปยังไดเรคทอรีที่ระบุ
+
+`clamdscan --move {{path/to/quarantine_directory}}`
+
+- ลบไฟล์ที่พบการติดไวรัส
+
+`clamdscan --remove`
+
+- ใช้มัลติเธรดในการตรวจไดเรคทอรี
```
Comment (kbdharun) [inline pages.th/common/clamdscan.md:30]:
```suggestion
- ใช้มัลติเธรดในการตรวจไดเรคทอรี:
```

File: pages.th/common/clamdscan.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`clamdscan --multiscan`
+
+- ส่งตัวอธิบายไฟล์ไปยังโปรแกรมแสกนพื้นหลังแทนการส่งไฟล์ข้อมูล
```
Comment (kbdharun) [inline pages.th/common/clamdscan.md:34]:
```suggestion
- ส่งตัวอธิบายไฟล์ไปยังโปรแกรมแสกนพื้นหลังแทนการส่งไฟล์ข้อมูล:
```

File: pages.th/linux/clamav.md (hunk 1 above)
Comment (navarroaxel) [reply pages.th/linux/clamav.md:1]: :thinking: Instead of the edition of the `clamav` page, translate `clamscan`. Let's work on `clamav` in another PR.

Comment (nontw) [reply pages.th/linux/clamav.md]: Certainly.  This is done.  I have also removed clamav's Thai translation.

**Comment by nontw** [comment]
Incorporate all the suggestions.  Didn't accidentally remove `:` this time :-)

File: pages.th/common/clamdscan.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+`{{command}} | clamdscan -`
+
+- แสกนไดเรคทอรีปัจจุบันแล้วแสดงผลไฟล์ที่ตรวจพบความบกพร่อง:
+
+`clamdscan --infected`
+
+- ส่งผลการแสกนไปยังไฟล์ที่ระบุ:
+
+`clamdscan --log {{path/to/log_file}}`
```
Comment (navarroaxel) [inline pages.th/common/clamdscan.md:20]: pla
Please translate the placeholders, like `path/to/quarantine_directory`, `path/to/log_file`, etc., for get this easier for non-English speakers.

**Comment by navarroaxel** [comment]
@nontw we have fixed the ClamAV pages in English version: 
- https://github.com/tldr-pages/tldr/blob/main/pages/common/clamav.md
- https://github.com/tldr-pages/tldr/blob/main/pages/common/clamdscan.md
- https://github.com/tldr-pages/tldr/blob/main/pages/common/clamscan.md
- https://github.com/tldr-pages/tldr/blob/main/pages/common/freshclam.md

Feel free to update this PR to include the full ClamAV translation in Thai

File: pages.th/common/clamdscan.md (hunk 7 above)
Comment (nontw) [reply pages.th/common/clamdscan.md]: done

File: pages.th/common/clamdscan.md (hunk 4 above)
Comment (nontw) [reply pages.th/common/clamdscan.md]: Done.

**Comment by nontw** [comment]
> @nontw we have fixed the ClamAV pages in English version:
> 
> * https://github.com/tldr-pages/tldr/blob/main/pages/common/clamav.md
> * https://github.com/tldr-pages/tldr/blob/main/pages/common/clamdscan.md
> * https://github.com/tldr-pages/tldr/blob/main/pages/common/clamscan.md
> * https://github.com/tldr-pages/tldr/blob/main/pages/common/freshclam.md
> 
> Feel free to update this PR to include the full ClamAV translation in Thai

Yep, I added the translation for both Clamav and FreshClam.  Let me know if they're ok.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**